components/ProfilePage/SmartWalletContents.tsx

```diff
@@ -4,14 +4,28 @@ import { useProfileProvider } from "../../providers/ProfileContext"
 import getSmartWallet from "../../lib/getSmartWallet"
 import getProfileFormattedCollection, { ALLNFTS } from "../../lib/getProfileFormattedCollection"
 import Media from "../../shared/Media"
+import getDefaultProvider from "../../lib/getDefaultProvider"
+import Deploy6551AndMintDNAButton from "./Deploy6551AndMintButton"
 
 const SmartWalletContents = () => {
   const { cre8orNumber } = useProfileProvider()
   const [ownedNfts, setOwnedNfts] = useState([])
-
+  const [hasSmartWallet, setHasSmartWallet] = useState(true)
+  const provider = getDefaultProvider(process.env.NEXT_PUBLIC_TESTNET ? 5 : 1)
   useEffect(() => {
     const init = async () => {
-      const smartWalletAddress = await getSmartWallet(cre8orNumber)
+      const smartWalletAddress = await getSmartWallet(85)
+      //
```

Comment:
Do we absolutely have to use 85 to get the address of smart wallet? ❔
It feels like it's hardcoded somewhere. 💭
If so, please ignore this comment.

Reply:
I agree with ziad. If it is necessary to use a magic number, better to set it as a variable with a clear meaning variable name.

Reply:
my bad, it was late
updating now

Reply:
updated from `85` => `cre8orNumber`

<img width="868" alt="Screenshot 2023-08-09 at 8 11 59 AM" src="https://github.com/defientco/CRE8ORS-UI/assets/23249402/03f9d6a2-7f7f-43ff-ad0f-5a29753c8929">